config/routes.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root 'commerce#index'
```

Comment:
```suggestion
  root to: redirect('/commerce')
```

senhao ele nao vai redirecionar, vai continuar na root